### ISSUE TITLE
Read plugin settings from environment variables

### ISSUE THREAD

**Issue by talenc2** [alerta/docker-alerta#2]
Some problems faced

1) AMQP

error: Unknown option auto_start_request
error: Unknown option use_greenlets

I had to review lines of the file usr/local/lib/python2.7/dist-packages/kombu/transport/mongodb.py:

            # 'auto_start_request': True,
            # auto_start_request = options ['auto_start_request'],

            # use_greenlets = _detect_environment ()! = 'default'

For these option is deprecated

2) Twilio

I entered the container shell, cloned the repo contrib, performed 'python setup' for Twilio, and had to change the file:

/usr/local/lib/python2.7/dist-packages/alerta_twilio-0.1.0-py2.7.egg/twilio_sms.py

with my credentials!

It did not work, export the variables.

Then I needed to enable the alertad.conf.sh:

PLUGINS = ['AMQP', 'reject', 'twilio_sms']

AMQP_URL = 'mongodb: // localhost: 27017 / kombu'
AMQP_QUEUE = 'alerts'
AMQP_TOPIC = 'notify'

All changes always with stop / start container!

I just wanted to share it if someone tb go through it!

**Comment by satterly** [comment]
@talenc2 the plugins (for Twilio and others) should probably look for configuration settings in the shell environment rather than expecting the user to modify the python script directly. I'll keep this issue open to track this request.

**Comment by satterly** [comment]
1) fixed as per guardian/alerta#90

**Comment by satterly** [comment]
Twilio SMS plugin settings can now be configured as environment variables, like so:

```
export TWILIO_ACCOUNT_SID=AC5ef872f6da5a21de157d80997a64bd33
export TWILIO_AUTH_TOKEN=[AuthToken]
export TWILIO_TO_NUMBER=+14158141829
export TWILIO_FROM_NUMBER=+16518675309
```
or via the alertad.conf configuration file, like so:

```
TWILIO_ACCOUNT_SID = 'AC5ef872f6da5a21de157d80997a64bd33'
TWILIO_AUTH_TOKEN = '[AuthToken]'
TWILIO_TO_NUMBER = '+14158141829'
TWILIO_FROM_NUMBER = '+16518675309'
```

See https://www.twilio.com/docs/api/rest/sending-messages for more info
